packages/fether-electron/src/main/app/constants/index.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Disallows users from using Fether with a remote node.
+// SSH tunnels are still possible.
+//
+// References: See trustworthiness of 127.0.0.1 vs localhost: https://letsencrypt.org/docs/certificates-for-localhost/
+const TRUSTED_URLS = [
+  // Allow open 127.0.0.1 using http in dev mode
+  process.env.NODE_ENV === 'development'
+    ? 'http://127.0.0.1:3000'
+    : 'https://127.0.0.1:3000',
```

Comment:
Hmm Luke, can you explain why you kept `https://127.0.0.1:3000`? There's nothing running on `https://127.0.0.1:3000` in production, Electron serves a local .html file (see [here](https://github.com/paritytech/fether/blob/674e3bf3ba6f42c20a2e819e0c9024c60f3fa72a/packages/fether-electron/src/main/app/options/config/index.js#L11-L17)).

Reply:
No reason, it's unnecessary so I've now removed it